Engine/platform/ios/acplios.cpp

```diff
@@ -357,11 +357,7 @@ void AGSIOS::DisplayAlert(const char *text, ...) {
   vsnprintf(displbuf, 2000, text, ap);
   va_end(ap);
   
-  // TODO: Print this properly?
-  printf("%s", displbuf);
-  
-//  Debug::Printf(kDbgMsg_Info, text);
-  
+  Debug::Printf(kDbgMsg_Warn, displbuf);
```

Comment:
The safer form would be similar to one previously used with printf:
`Debug::Printf(kDbgMsg_Warn, "%s", displbuf);`
This is in case input contained formatting tokens.